cfgov/scripts/export_enforcement_actions.py

```diff
@@ -0,0 +1,108 @@
+from __future__ import unicode_literals
+
+import datetime
+from six.moves import html_parser as HTMLParser
+
+from django.http import HttpResponse
+from django.utils import html
+
+import unicodecsv
+
+import csv
+import re
+from v1.models import DocumentDetailPage
+from v1.util.migrations import get_stream_data
+
+
+
+html_parser = HTMLParser.HTMLParser()
+
+HEADINGS = ['Matter name', 'Date filed', 'URL', 'Status', 'Category', 'File number', 'Content', 'Preview text']
+
+
+def clean_and_strip(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(unescaped).strip()
+
+
+def assemble_output():
```

Comment:
It would be good to have tests — particularly for this function, because that would illustrate the data structure we're expecting here. I don't mind throwing something together for that since the bulk of this function is mine.